src/styles/layout/_header.scss

```diff
@@ -0,0 +1,44 @@
+.header-container {
+  font-family: "Quicksand", sans-serif;
+  width: 100%;
+  font-weight: 500;
+  height: 111px;
+  position: absolute;
+  z-index: 99;
+  background-color: transparent;
+
+  .container {
+    height: 100%;
+  }
+
+  .row {
```

Comment:
row là gì tạo sao set là 100%

Reply:
- row này có sẵn trong source ban đầu, set 100% để có thể ăn theo toàn bộ height của class header

Reply:
row đã có sẵn trong thư viện không cần custom thêm